.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,7 +8,7 @@ labels: bug
 ## Do this before creating an issue
 <!-- Delete this section before the submit -->
 
-- Check our [developer documentation](https://developers.line.me/en/docs/) and [FAQ](https://developers.line.me/en/faq/messaging-api/) page for more information on LINE bots and the Messaging API
+- Check our [developer documentation](https://developers.line.biz/en/docs/messaging-api/), [FAQ](https://developers.line.biz/en/faq/), and [community Q&A](https://www.line-community.me/questions) for more information on LINE bots and the Messaging API
```

Comment:
[ask]
- Is `https://developers.line.biz/en/docs/messaging-api/` correct? or just `https://developers.line.biz/en/docs/`?

[imo]
- `developers documentation`(or just `documentation`) is better than `developer documentation`
- `developers community` is better than `community Q&A`

Reply:
`https://developers.line.biz/en/docs/messaging-api/` is correct.
`https://developers.line.biz/en/docs/` includes other docs. (e.g. LINE Login)